crates/sui-framework/packages/dwallet-system/sources/dwallet.move

```diff
@@ -302,6 +326,32 @@ module dwallet_system::dwallet {
             signatures,
             sender: session.sender,
         };
-        transfer::transfer(sign_output, session.sender);
+        // TODO: Change output to be immutable instead of owned
```

Comment:
```suggestion
```
Fix TODOs before submitting a PR

Reply:
Fixing this todo will require me to update the tests, as it depends on this object being owned for the fetch. Do you want me to do this in this PR?

Reply:
I added this TODO to existing code, I don't think it makes sense to change it in this PR. I created an issue and linked this todo to the issue number.